code/infrastructure-api/src/credentials/zeppelinShiroGenerator.js

```diff
@@ -29,7 +29,7 @@ function generateHash(password, salt) {
   try {
     let hashVal = calculateDigest(salt + password);
     for (let i = 1; i < ITERATIONS; i += 1) {
-      hashVal = calculateDigest(new Buffer(hashVal, 'hex'));
+      hashVal = calculateDigest(Buffer.from(hashVal, 'hex'));
```

Comment:
Are there existing tests for this? Once we get a deployed environment we need to check this still works :-)

Reply:
There are